docs/194-inicjalizacja.md

```diff
@@ -0,0 +1,87 @@
+# Inicjalizacja bota
+  
+## Punkt wejściowy
+
+Jeśli uruchamiamy bota lokalnie, punktem wejściowym jest plik  "Program" w projekcie "Watchman.Discord". Tutaj
```

Comment:
brakuje mi tutaj enterów
przykładowo

`Jeśli uruchamiamy bota lokalnie, punktem wejściowym jest plik  "Program" w projekcie "Watchman.Discord". Tutaj`
Przed "Tutaj" powinien być enter

Aktualnie wygląda to tak, że tekst się zlewa